proto/core/auth/signed_message.proto

```diff
@@ -1,9 +1,12 @@
 syntax = "proto3";
 package syft.core.auth;
 
+import "proto/core/common/common_object.proto";
+
 message SignedMessage {
-  string obj_type = 1;
-  bytes signature = 2;
-  bytes verify_key = 3;
-  bytes message = 4;
+  syft.core.common.UID msg_id = 1;
```

Comment:
I'm surprised we needed to add this here. Is the ID different from the id of the message inside SignedMessage?

Reply:
Well - maybe not too surprised. It makes sense we'd wnat ot have access to the ID without having to deserialize message. I'm hoping that the msg id isn't different than message though.

Reply:
The idea was to keep them the same which means as messages flow around you can keep track of them regardless of whether they are signed or not. Thats how its working now, but if its wrong please let me know. I wasn't sure if we had a requirement that all messages had unique IDs since in a way the contents and a signed message are not ==.